zzansuni-api-server/app/src/main/resources/db/migration/V4__add_column_review.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE challenge_review ADD COLUMN difficulty BIGINT NOT NULL;
+ALTER TABLE challenge_review ADD COLUMN achievement BIGINT NOT NULL;
```

Comment:
```suggestion
ALTER TABLE challenge_review ADD COLUMN difficulty TINYINT DEFULT(1) NOT NULL;
ALTER TABLE challenge_review ADD COLUMN achievement TINYINT DEFULT(1) NOT NULL;

ALTER TABLE challenge_review ALTER COLUMN difficulty DROP DEFAULT;
ALTER TABLE challenge_review ALTER COLUMN achievement DROP DEFAULT;
```

기존 더미데이터 고려해서 default 0 넣어주세용
그리고 BIGINT를 안해도 되지않을까요 ?!